scripts/release/update-spm-manifest.sh

```diff
@@ -4,52 +4,34 @@ set -euo pipefail
 
 #
 # Usage:
-#   ./scripts/release/update-spm-manifest.sh <maps version number> <common version number> <core version number> <maps xcframework checksum>
+#   ./scripts/release/update-spm-manifest.sh <maps version number> <maps xcframework checksum> <common version number> <core version number>
 #
 
 MAPS_VERSION=${1}
-COMMON_VERSION=${2//v/}
-CORE_VERSION=${3//v/}
-CHECKSUM=${4}
-GITHUB_TOKEN=$(./scripts/get_token.js)
+CHECKSUM=${2}
+COMMON_VERSION=${3//v/}
+CORE_VERSION=${4//v/}
+
+#
+# Checkout the release branch
+#
+git fetch origin main
+git checkout Release/v${MAPS_VERSION}
 
 # Update Package.swift
 sed -i '' s/"let version = \".*\""/"let version = \"${MAPS_VERSION}\""/ Package.swift
 sed -i '' s/"let checksum = \".*\""/"let checksum = \"${CHECKSUM}\""/ Package.swift
 sed -i '' s/"mapbox-common-ios.git\", .exact(\".*\")"/"mapbox-common-ios.git\", .exact(\"${COMMON_VERSION}\")"/ Package.swift
 sed -i '' s/"mapbox-core-maps-ios.git\", .exact(\".*\")"/"mapbox-core-maps-ios.git\", .exact(\"${CORE_VERSION}\")"/ Package.swift
 
+# Update MapboxMaps.podspec
+sed -i '' s/"maps_version = '.*'"/"maps_version = '${MAPS_VERSION}'"/ MapboxMaps.podspec
+sed -i '' s/"m.dependency 'MapboxCommon', '.*'"/"m.dependency 'MapboxCommon', '${COMMON_VERSION}'"/ MapboxMaps.podspec
+sed -i '' s/"m.dependency 'MapboxCoreMaps', '.*'"/"m.dependency 'MapboxCoreMaps', '${CORE_VERSION}'"/ MapboxMaps.podspec
+
 #
```

Comment:
Is there any way we could test the validity of these changes, before the commit & push?

Reply:
We are on a release branch, so the last step of our CI is to make these changes in automation and commit/push. 

Part of the process is for the releaser to validate these changes before tagging